integration_tests/run_pyspark_from_build.sh

```diff
@@ -28,7 +28,7 @@ else
     # support alternate local jars NOT building from the source code
     if [ -d "$LOCAL_JAR_PATH" ]; then
         CUDF_JARS=$(echo "$LOCAL_JAR_PATH"/cudf-*.jar)
-        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark*.jar)
+        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
```

Comment:
fix file name filter

Reply:
Why was this causing issues where it does not cause issues in other places? It would be good to add a comment explaining this because a few lines below it in the else clause.